code/modules/mob/living/carbon/xenomorph/XenoAttacks.dm

```diff
@@ -100,7 +100,7 @@
 	if(HAS_TRAIT(src, TRAIT_ABILITY_BURROWED))
 		return XENO_NO_DELAY_ACTION
 
-	if(islarva(M)) //Larvas can't eat people
+	if(islarva(M) || islesserdrone(M)) //Larvas can't eat people, now neither can Lessers
```

Comment:
uh what ? this makes lessers unable to ATTACK

Reply:
Overlooked that. Sorry.
Fixed.